snet_cli/mpe_service_command.py

```diff
@@ -171,6 +174,32 @@ def print_service_metadata_from_registry(self):
         metadata      = self._get_service_metadata_from_registry()
         self._printout(metadata.get_json_pretty())
 
+    def _service_status(self, url, secure=True):
+        channel = None
+        try:
+            channel = open_grpc_channel(endpoint=url)
+            stub = heartb_pb2_grpc.HealthStub(channel)
+            try:
```

Comment:
Lets add a timeout for the healthcheck as well. We can go with 15 seconds

Reply:
added timeout as 10 sec same as in Dapp